docs/LANGUAGES.md

```
@@ -41,11 +41,14 @@ translations:
 
 ## How to ADD a new string
 
-1. Add it to `lng_tmpl/_base.yml`:
+1. Add it to the **END** of `lng_tmpl/_base.yml`, below the APPEND-ONLY banner (~line 923):
    ```yaml
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)